tree/tree/inc/TTree.h

```diff
@@ -460,7 +460,7 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    virtual Long64_t        GetEntriesFast() const   { return fEntries; }
    virtual Long64_t        GetEntriesFriend() const;
    virtual Long64_t        GetEstimate() const { return fEstimate; }
-   virtual Int_t           GetEntry(Long64_t entry = 0, Int_t getall = 0);
+   virtual Int_t           GetEntry(Long64_t entry, Int_t getall = 0);
            Int_t           GetEvent(Long64_t entry = 0, Int_t getall = 0) { return GetEntry(entry, getall); }
```

Comment:
If we update `GetEntry` we should probably also update `GetEvent`

Reply:
Okay. Does that mean I should reopen this PR?